Docs: Update license

UlisesGascon opened the pull request:
## Main changes

Updated license references to MIT

## Context

Related https://github.com/onebeyond/admin/issues/27

cressie176 commented on LICENSE.txt:
```diff
@@ -1,15 +1,21 @@
+MIT License
 
-Copyright (c) 2016, GuideSmiths Ltd.
+Copyright (c) One Beyond 2022 to present.
```
I suggest

Copyright 2016-present, GuideSmiths Ltd. Assigned to One Beyond

inigomarquinez replied:
Perhaps we can make it a little more clear taking into account example of other projects that have been moved between different organizations, such as [express](https://github.com/expressjs/express/blob/master/LICENSE):

```
Copyright (c) 2009-2014 TJ Holowaychuk <tj@vision-media.ca>
Copyright (c) 2013-2014 Roman Shtylman <shtylman+expressjs@gmail.com>
Copyright (c) 2014-2015 Douglas Christopher Wilson <doug@somethingdoug.com>
```

In our case would be something such as:

```
Copyright (c) 2016-2022 GuideSmiths Ltd.
Copyright (c) 2013-present One Beyond
```

And add an email if we have it.

Betisman replied:
There's a typo in your suggestion @inigomarquinez 
```suggestion
Copyright (c) 2016-2022 GuideSmiths Ltd.
Copyright (c) 2023-present One Beyond
```

Isn't there an official way to state these changes?
I've read about both approaches of just setting the ranges or adding something like `date of copyright transferral`. So I am not sure which one is the best.

UlisesGascon commented:
Thanks for the feedback and suggestions :)

Are we  good to go with this change proposed by @Betisman in [71955ec](https://github.com/onebeyond/rascal/pull/214/commits/71955ecfcb45c56a9bdbe2b5133305ba4783a0c0)?

UlisesGascon commented:
I fixed the years typo in ([0ab8f56](https://github.com/onebeyond/rascal/pull/214/commits/0ab8f562fc9fd974d0670beb574c160129ea7d95)) and updated the branch with the last changes from `main` ([ee1b6d1](https://github.com/onebeyond/rascal/pull/214/commits/ee1b6d1456d70f506e5e4f5ec583a46aaaf16d2d))
